webapp/src/ts/components/snackbar/snackbar.component.ts

```diff
@@ -67,6 +71,21 @@ export class SnackbarComponent implements OnInit, OnDestroy {
     this.subscription.unsubscribe();
   }
 
+  private subscribeToRoute() {
+    const subscription = this.router.events
+      .pipe(
+        delay(500), // Waiting for FAB component to render
+        filter(event => event instanceof NavigationEnd),
+      ).subscribe(() => {
+        if (!this.active) {
+          return;
+        }
+        this.displayAboveFab = this.isFABDisplayed();
+        this.changeDetectorRef.detectChanges(); // Running outside Angular's zone, so calling detectChanges
```

Comment:
[Running outside Angular's zone and the component is working with this](https://github.com/medic/cht-core/commit/03d8dc434047d0f87509911ca0e1fd6685920448#diff-02ba99a9dbe8d57d75a8b452b0fd29918280b96eaddb33507bedd37a66e8846f)